src/Azure.Functions.Cli/Actions/LocalActions/CreateFunctionAction.cs

```diff
@@ -75,6 +76,13 @@ public async override Task RunAsync()
             }
 
             var workerRuntime = GlobalCoreToolsSettings.CurrentWorkerRuntimeOrNone;
+            if (!FileSystemHelpers.FileExists(Path.Combine(Environment.CurrentDirectory, "local.settings.json")))
+            {
+                // we're assuming "func init" has not been run
+                await InitApp(workerRuntime);
+                workerRuntime = WorkerRuntimeLanguageHelper.GetCurrentWorkerRuntimeLanguage(_secretsManager);
```

Comment:
I'm concerned that this is the correct way to update `workerRuntime`. My worry is that `GlobalCoreToolsSettings` will continue to return `None` for runtime unless we run `Init` again, but as far as I can see we no longer have access to the `args` variable. Do we need to run `Init` with blank args or is that a concern?